src/test/java/uk/gov/digital/ho/proving/income/validator/IncomeValidatorTestIT.java

```diff
@@ -22,13 +24,7 @@
 
 @RunWith(SpringRunner.class)
 @SpringBootTest(classes = {
-    IncomeValidationService.class,
-    CatASalariedIncomeValidator.class,
-    CatASalariedMonthlyIncomeValidator.class,
-    CatASalariedWeeklyIncomeValidator.class,
-    CatAUnsupportedIncomeValidator.class,
-    CatBNonSalariedIncomeValidator.class,
-    EmploymentCheckIncomeValidator.class
+    ServiceRunner.class
```

Comment:
If you want to bring up the entire application I believe you can omit classes entirely... I'm unclear of the pros for doing so, can we discuss?